Mathlib/Algebra/Lie/Submodule.lean

```diff
@@ -528,16 +529,23 @@ instance : IsModularLattice (LieSubmodule R L M) where
 
 variable (R L M)
 
+/-- The natural functor that forgets the action of `L` as an order embedding. -/
+@[simps] def toSubmodule_orderEmbedding : LieSubmodule R L M ↪o Submodule R M :=
```

Comment:
Nit: could use `where` notation

Reply:
But doesn't `where` produce terms containing these horrible `let src := ...` clauses?

Reply:
No, it's `with`/ `__ :=` that does that.

Reply:
But we have no evidence that those `let`s are problematic for *morphisms*, as far as I know; the problems arise mainly for instances.